Lib/fontTools/ttLib/ttFont.py

```diff
@@ -755,6 +755,10 @@ def draw(self, pen):
 		"""
 		self._glyph.draw(pen)
 
+	def drawPoints(self, pen):
+		# drawPoints is only implemented for _TTGlyphGlyf at this time.
+		raise NotImplementedError
```

Comment:
```suggestion
		raise NotImplementedError()
```

see https://www.python.org/dev/peps/pep-0317/

Reply:
or you can put that inline comment as the message for the NotImplementedError exception, even better